lib/tasks/publishing_api.rake

```diff
@@ -1,5 +1,5 @@
 namespace :publishing_api do
-  desc "Send pages to the content-api"
+  desc "Send pages to the Content Store"
```

Comment:
Interestingly, it looks like this code has actually always sent content to the Publishing API [1], and I think that would make a better description, as while the content may have ended up in the content-api, and will end up in the content store, sending it to the Publishing API is what the task actually does.

1: https://github.com/alphagov/calculators/commit/cae9720711c4ad73e6bca4ec9ac3fe8dc89602cd